src/_buttons.scss

```diff
@@ -225,3 +225,78 @@
     }
   }
 }
+
+
+.button.is-icon-only-button {
+  background-color: transparent;
+  color: lighten($button-icon-only-default-color, 10.5%);
```

Comment:
Why 10.5%? It's a magic number whatever we do here, but does the extra 0.5 make much difference from straight 10%?

Reply:
It's no difference, probably the leftover of some prior value/copy+paste. Fixed in 432c73.